pkgs/sketch_pad/lib/keys.dart

```diff
@@ -32,13 +32,19 @@ const ShortcutActivator codeCompletionKeyActivator = SingleActivator(
   LogicalKeyboardKey.space,
   control: true,
 );
+final ShortcutActivator quickFixKeyActivator = SingleActivator(
+  LogicalKeyboardKey.period,
+  meta: _mac,
```

Comment:
I was surprised by this so I looked it up and it's VS Code's default so it makes sense. It's just very different from what I'm used to :)

Reply:
It's also different from what DartPad uses today...

Ultimately we just want something familiar and discoverable. The VS Code bindings seem like a reasonable choice. I'm curious - what binding would you expect here?

Reply:
Option+enter on macOS since that's the IntelliJ default.

I think you're right though for a web-based UI the VS code default bindings are a good choice, especially with more and more online editors being monaco/Code-OSS based.